app/models/host_status/build_status.rb

```diff
@@ -56,10 +56,6 @@ def to_status(options = {})
       end
     end
 
-    def relevant?(options = {})
```

Comment:
Should we just deprecate and return `true` in the meantime?

Reply:
We do return true, from the base class definition. Other status have different definitions for `relevant?`